src/extension/gogoduck/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule.java

```diff
@@ -0,0 +1,176 @@
+package au.org.emii.gogoduck.worker;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.URL;
+import java.net.URLEncoder;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ucar.nc2.Attribute;
+import ucar.nc2.NetcdfFile;
+import ucar.nc2.NetcdfFileWriter;
+
+import au.org.emii.gogoduck.worker.GoGoDuck.UserLog;
+
+public class GoGoDuckModule {
+    private static final Logger logger = LoggerFactory.getLogger(GoGoDuckModule.class);
+
+    protected String profile = null;
+    protected String geoserver = null;
+    protected SubsetParameters subset = null;
+    protected UserLog userLog = null;
+
+    public GoGoDuckModule() {}
```

Comment:
Yeah, this is what I said. Java allows you to do it, but the code will be ugly. I tried it when writing the GN link monitor, then looked at it again and was like "WTF IS THAT?!", quickly factored it into a simple `init()` function. I think by now I completely removed that code, but tell you what, it did not look good :)